Dockerfile-centos

```diff
@@ -0,0 +1,36 @@
+FROM centos:7
+MAINTAINER Bryan Latten <latten@adobe.com>
+
+# Use in multi-phase builds, when an init process requests for the container to gracefully exit, so that it may be committed
+# Used with alternative CMD (worker.sh), leverages supervisor to maintain long-running processes
+ENV SIGNAL_BUILD_STOP=99 \
+    S6_BEHAVIOUR_IF_STAGE2_FAILS=2 \
+    S6_KILL_FINISH_MAXTIME=5000 \
+    S6_KILL_GRACETIME=3000 \
+    S6_VERSION=v1.18.1.5 \
+    GOSS_VERSION=v0.2.5
+
+# Ensure scripts are available for use in next command
+# COPY ./container/root/security_updates.sh / \
+#      ./container/root/clean.sh /
+
+# - Upgrade base security packages, then clean packaging leftover
+# - Add S6 for zombie reaping, boot-time coordination, signal transformation/distribution
+# - Add goss for local, serverspec-like testing
+RUN curl -L https://github.com/just-containers/s6-overlay/releases/download/${S6_VERSION}/s6-overlay-amd64.tar.gz -o /tmp/s6.tar.gz && \
+    tar xzf /tmp/s6.tar.gz -C / && \
+    rm -f /tmp/s6.tar.gz && \
+    curl -L https://github.com/aelsabbahy/goss/releases/download/${GOSS_VERSION}/goss-linux-amd64 -o /usr/local/bin/goss && \
+    chmod +x /usr/local/bin/goss && \
+    # Sync common locations of bash and shell between container variants
+    ln -s /usr/bin/sh /bin/sh && \
+    ln -s /usr/bin/bash /bin/bash
```

Comment:
on CentOS, at least on a regular install, usually /bin is already a symlink to /usr/bin

i.e.

```
$ ll /bin
lrwxrwxrwx 1 root root 7 Oct  4 15:23 /bin -> usr/bin
```